scripts/publish-deb-to-bintray.sh

```diff
@@ -20,7 +25,7 @@ if [ -z "$BINTRAY_API_KEY" ]; then
 fi
 
 setVersion () {
-  VERSION=$(curl --silent "https://api.github.com/repos/stripe/stripe-cli/releases/latest" | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/');
+  VERSION=$(curl --silent "https://api.github.com/repos/stripe/stripe-cli/releases/latest" -u $GITHUB_TOKEN: | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/');
```

Comment:
I've updated the .deb and .rpm release scripts to authenticate requests to GitHub. Since we use unauthenticated requests to check for new versions from the CLI itself, we can easily run into rate limits, which would cause the scripts to fail.